system-tests/test/deposit-redemption.test.ts

```diff
@@ -1,64 +1,55 @@
-import { computeHash160, createAddressFromPublicKey } from "@keep-network/tbtc-v2.ts/dist/src/bitcoin"
-import { Hex } from "@keep-network/tbtc-v2.ts/dist/src/hex"
+  EthereumTBTCToken,
+  EthereumTBTCVault,
+  DepositFunding,
+  DepositScript,
+  BitcoinAddressConverter,
+  WalletTx,
+  EthereumWalletRegistry,
+  BitcoinHashUtils,
+  EthereumAddress,
+  TBTC,
+} from "@keep-network/tbtc-v2.ts"
 import { BigNumber, constants, Contract } from "ethers"
 import chai, { expect } from "chai"
-import { submitDepositTransaction } from "@keep-network/tbtc-v2.ts/dist/src/deposit"
-import { submitDepositSweepTransaction } from "@keep-network/tbtc-v2.ts/dist/src/deposit-sweep"
-import { submitRedemptionTransaction } from "@keep-network/tbtc-v2.ts/dist/src/redemption"
 import chaiAsPromised from "chai-as-promised"
 
 import { setupSystemTestsContext } from "./utils/context"
-import { generateDeposit } from "./utils/deposit"
 import { fakeRelayDifficulty, waitTransactionConfirmed } from "./utils/bitcoin"
 
-import type { UnspentTransactionOutput } from "@keep-network/tbtc-v2.ts/dist/src/bitcoin"
+import type {
+  RedemptionRequest,
+  BitcoinUtxo,
+  TBTCContracts,
+  DepositReceipt,
+} from "@keep-network/tbtc-v2.ts"
 import type { SystemTestsContext } from "./utils/context"
-import type { RedemptionRequest } from "@keep-network/tbtc-v2.ts/dist/src/redemption"
-import type { Deposit } from "@keep-network/tbtc-v2.ts/dist/src/deposit"
-import { TBTCToken } from "@keep-network/tbtc-v2.ts/dist/src/ethereum"
 
 chai.use(chaiAsPromised)
 
-/**
- * This system test scenario performs a single deposit and redemption.
- *
- * The scenario consists of the following steps:
- * 1. The depositor broadcasts the deposit transaction on BTC chain and reveals
- *    it to the bridge.
- * 2. The wallet broadcasts the sweep transaction of the given deposit on BTC
- *    chain and submits the sweep proof to the bridge.
- * 3. The depositor (redeemer) requests the redemption of its entire bank
- *    balance.
- * 4. The wallet broadcasts the redemption transaction handling the given
- *    request and submits the redemption proof to the bridge.
- *
- * Following prerequisites must be fulfilled to make a successful pass:
- * - The depositor's BTC balance must allow to perform the deposit
- * - tBTC v2 contracts must be deployed on used Ethereum network
- * - A fresh live wallet (with no main UTXO yet) must be registered in
- *   the bridge
- */
 describe("System Test - Deposit and redemption", () => {
   let systemTestsContext: SystemTestsContext
-  let electrumClient: ElectrumClient
+
   let tbtcTokenAddress: string
   let bridgeAddress: string
   let vaultAddress: string
-  let tbtcTokenHandle: TBTCToken
-  let maintainerBridgeHandle: EthereumBridge
-  let depositorBridgeHandle: EthereumBridge
+  let walletRegistryAddress: string
+
   let bank: Contract
   let relay: Contract
   let tbtc: Contract
 
+  let electrumClient: ElectrumClient
```

Comment:
I would abandon this reference in favor of using the `bitcoinClient` from the SDK instance. This way, we are not attaching system tests to any specific Bitcoin client implementation.

Reply:
Done in 3c4fd74455625a9acd2952991aa6da3db517ad5a.